pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: "Migrating from GOV.UK PaaS"
+---
+
+# Migrating from GOV.UK PaaS
+
+## Last updated
+
+- 1st November 2022 - first published
+
+## Government cloud hosting strategy
+
+The Government Digital Service (GDS) has provided GOV.UK PaaS since 2015, supporting and providing a public cloud platform for departments using a shared hosting and responsibility model. 
+
+Following an extensive analysis period, GDS has concluded that, while the platform has been successful in its aims, the underlying technology would now require investment before it could meet its goals in the long term. 
+
+Faced with this need for re-investment GDS has decided to [decommission the platform](https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/), in order to focus its budget and energy on other GDS products for common use by Government.
+
+GDS will not be providing a replacement hosting service.
+
+Tenants should look to their own department, or parent department, and use existing patterns and support available for public cloud hosting services. 
```

Comment:
Where are these existing patterns? They will be asked for and they need to be public.

Reply:
Paul asked Phil a few times for something to link to here, suggest we remove the second half of the sentence while this is still unclear.

Reply:
ok I have trimmed the sentence